libqtile/backend/wayland/window.py

```diff
@@ -444,11 +444,9 @@ def _reconfigure_floating(self, x, y, w, h, new_float_state=FloatStates.FLOATING
                 self.group.mark_floating(self, True)
             hook.fire('float_change')
 
-    def cmd_focus(self, warp=None):
+    def cmd_focus(self, warp: bool = True) -> None:
         """Focuses the window."""
-        if warp is None:
-            warp = self.qtile.config.cursor_warp
-        self.focus(warp=warp)
```

Comment:
I'm not sure to understand this diff and the next one. Isn't setting warp to `True` by default and removing the `config.cursor_warp` check going to make cursor warping enabled for everyone?

Reply:
Nope - `warp` isn't so much "warp" as it is "now would be a good time to warp if `cursor_warp` is enabled". See line 922 - `if warp and self.qtile.config.cursor_warp` - warping will _never_ happen if `config.cursor_warp` is False. The current version is checking `config.cursor_warp` two times.